scapy/layers/ppp.py

```diff
@@ -463,24 +501,30 @@ def dispatch_hook(cls, _pkt=None, *args, **kargs):
 
 class PPP_LCP_MRU_Option(PPP_LCP_Option):
     fields_desc = [ByteEnumField("type", 1, _PPP_lcp_optiontypes),
-                   FieldLenField("len", 4, fmt="B", adjust=lambda p, x:4),
+                   ByteField("len", 4),
                    ShortField("max_recv_unit", 1500)]
 
 
-_PPP_LCP_auth_protocols = {0xc023: "Password authentication protocol",
-                           0xc223: "Challenge-response authentication protocol",  # noqa: E501
-                           0xc227: "PPP Extensible authentication protocol"}
+_PPP_LCP_auth_protocols = {
+    0xc023: "Password authentication protocol",
+    0xc223: "Challenge-response authentication protocol",
+    0xc227: "PPP Extensible authentication protocol",
+}
 
-_PPP_LCP_CHAP_algorithms = {5: "MD5",
-                            6: "SHA1",
-                            128: "MS-CHAP",
-                            129: "MS-CHAP-v2"}
+_PPP_LCP_CHAP_algorithms = {
+    5: "MD5",
+    6: "SHA1",
+    128: "MS-CHAP",
+    129: "MS-CHAP-v2",
+}
 
 
 class PPP_LCP_ACCM_Option(PPP_LCP_Option):
-    fields_desc = [ByteEnumField("type", 2, _PPP_lcp_optiontypes),
-                   FieldLenField("len", 6, fmt="B"),
-                   BitField("accm", 0x00000000, 32)]
+    fields_desc = [
+        ByteEnumField("type", 2, _PPP_lcp_optiontypes),
+        FieldLenField("len", 6, fmt="B"),
```

Comment:
Does that do anything ?

Reply:
Nope, fixed (I missed this one while fixing the others).